src/DittoMachine.sol

```diff
@@ -45,7 +46,7 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
     uint256 public constant BASE_TERM = 2**18;
     uint256 public constant MIN_FEE = 32;
     uint256 public constant DNOM = 2**16 - 1;
-    uint256 public constant MIN_AMOUNT_FOR_NEW_CLONE = BASE_TERM + (BASE_TERM * MIN_FEE / DNOM);
+    uint256 public constant MIN_AMOUNT_FOR_NEW_CLONE = BASE_TERM + ((BASE_TERM * MIN_FEE / DNOM));
```

Comment:
redundant double parenthesis

Reply:
ah, yes. there's probably going to be a few of these cause i was experimenting with some math things